addon/components/search-help-modal/component.js

```diff
@@ -15,7 +15,7 @@ export default Ember.Component.extend({
     layout,
     isOpen: false,
     currentPath: Ember.computed(function() {
-        return Ember.getOwner(this).lookup('controller:application').currentPath;
+        return window.location.origin + window.location.pathname;
```

Comment:
I'd recommend against using a computed property and instead set it in one of the [lifecycle hooks](https://guides.emberjs.com/v2.8.0/components/the-component-lifecycle/). You also may want to use window.location.href, or prefer template strings instead of the + operator.